package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.3.0",
   "license": "Apache License 2.0",
   "config": {
-    "webservice_version": "1.10.0-alpha.0"
+    "webservice_version": "https://5092-33383826-gh.circle-artifacts.com/0/tmp/artifacts/dockstore-webservice-1.10.0-alpha.1-SNAPSHOT.jar"
```

Comment:
should be something like https://app.circleci.com/pipelines/github/dockstore/dockstore/4188/workflows/9c62995e-d9e5-4b23-bf51-37c6bdc8c12f/jobs/4674/artifacts which gives more information

Reply:
.

Reply:
Rebased off develop with Natalie's changes and I'm just using that.